src/main/java/org/example/service/DeveloperService.java

```diff
@@ -1,28 +1,27 @@
 package org.example.service;
 
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonMappingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
 import org.example.model.Developer;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import java.sql.Connection;
-import java.sql.ResultSet;
-import java.sql.SQLException;
-import java.sql.Statement;
+import java.sql.*;
 import java.util.ArrayList;
 import java.util.List;
 
 public class DeveloperService {
 
-    private static final String GET_DEVELOPERS = "select id, firstname, lastname, age, proglang from developer";
+    private static final String GET_DEVELOPERS = "select id, firstName, lastName, age, programmingLanguage from developer";
+    private static final String ADD_DEVELOPER = "INSERT INTO developer(firstName, lastName, age, programmingLanguage) VALUES (?, ?, ?, ?)";
     private static final String COLUMN_ID = "id";
-    private static final String COLUMN_FIRST_NAME = "firstname";
-    private static final String COLUMN_LASTNAME = "lastname";
-    private static final String COLUMN_AGE = "age";
-    private static final String COLUMN_PROG_LANG = "proglang";
-
+    public static final String COLUMN_FIRST_NAME = "firstName";
```

Comment:
Why there is a duplication of fields?
The same thing exists in the `Developer` class.